front/src/modules/pathfinding/hook/useInfra.ts

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect } from 'react';
+
+import { isEqual } from 'lodash';
+import { useSelector } from 'react-redux';
+
+import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import { useOsrdConfSelectors } from 'common/osrdContext';
+
+export default function useInfra() {
```

Comment:
As we will have (in this [PR](https://github.com/OpenRailAssociation/osrd/pull/7904)) a new useInfra.ts custom hook in modules/infra (to just fetch the infra), could you rename this one (and the file) so we don't make any mistake between the two ? Suggestions : useInfraStatus, useInfraLoadingState, useManageInfra...